.devcontainer/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2022 Robert Bosch GmbH and Microsoft Corporation
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+FROM ghcr.io/eclipse-velocitas/devcontainer-base-images/python:v0.1.3
```

Comment:
We need feature "docker-in-docker", unfortunately I cannot build it up locally due to proxy... If you can assist me how to avoid it - I am free for suggestions  :)

Reply:
why do you need docker-in-docker? you don't need to start the base-image in the devcontainer or?

Reply:
I need to operate with docker buildx. It is also required docker on board

Reply:
but docker is something else than docker in docker

Reply:
with latest commit ubuntu:22.04 was used as baseimage